### PR TITLE
Name export file with model name and date

### PR DESCRIPTION
**Pull request by anthonydillon**

### CODE REVIEW

**Comment by anthonydillon** [comment]
## QA
- Add some services and commit them
- Export the bundle and check the file name consists of `{environment name}-{YYYY}-{MM}-{DD}.yaml`
- Change environment and export
- Check the file name is correct

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5108/
Test PASSed.

File: jujugui/static/gui/src/app/views/utils.js [hunk 1]
```diff
@@ -1207,7 +1207,23 @@ YUI.add('juju-view-utils', function(Y) {
...
+    saveAs(exportBlob, this._getExportName(envName));
+  },
+
+  /**
+    Get the export file name
+
+    @method _getExportName
+    @param {String} Enviroment name
+  */
+  utils._getExportName = function(envName) {
```
Comment (hatched) [inline jujugui/static/gui/src/app/views/utils.js]: `_genereateBundleExportFileName`?

Comment (hatched) [inline jujugui/static/gui/src/app/views/utils.js]: To make this testable:

`utils._generateBundleExportFileName = function(envName, date=new Date()) {`

Then in the test you can pass in your own custom date object to get the same output.

**Comment by huwshimi** [comment]
Nice little change @anthonydillon. :+1: with the changes from @hatched + a test. Thanks!

**Comment by kadams54** [comment]
QA is :ok:

**Comment by anthonydillon** [comment]
Thanks for the points. Done them and added a test. Ready for review again.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5115/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5116/
Test PASSed.

**Comment by hatched** [comment]
Thanks ant this looks good!

**Comment by kadams54** [comment]
QA, round 2, is :ok: - nice work @anthonydillon!

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**